### ISSUE TITLE
standard_init_linux.go:228: exec user process caused: no such file or directory on Windows

### ISSUE THREAD

**Issue by marcdexet-cnrs** [giovtorres/slurm-docker-cluster#19]
# Context

* Windows 10
* Docker version 20.10.12, build e91ed57
* Docker Compose version v2.2.3
* Commit 3e6edfd8becbc97570eaee1484540cd2af2e56af

# Steps to reproduce

* Clone project
* run image creation `docker build -t slurm-docker-cluster:19.05.1 .`
* run `docker-compose up`
* See following error

```
slurmdbd   | standard_init_linux.go:228: exec user process caused: no such file or directory
slurmdbd exited with code 1
slurmctld  | standard_init_linux.go:228: exec user process caused: no such file or directory
slurmctld exited with code 1
c2         | standard_init_linux.go:228: exec user process caused: no such file or directory
c1         | standard_init_linux.go:228: exec user process caused: no such file or directory
```

**Comment by marcdexet-cnrs** [comment]
I found out the solution and applied it

See https://stackoverflow.com/questions/68881023/docker-standard-init-linux-go228-exec-user-process-caused-no-such-file-or-dir

* I converted all files from CRLF to LF, 
* I rebuild all images from scratch

Now it's OK.

**Comment by marcdexet-cnrs** [comment]
May a `.gitattribute` file be the solution

https://stackoverflow.com/a/42135910

**Comment by marcdexet-cnrs** [comment]
For windows users who encounter problems as 

```
slurm-dbd  | -- Waiting for database to become active ...
' (2)-dbd  | ERROR 2005 (HY000): Unknown MySQL server host 'mysql
slurm-dbd  | -- Waiting for database to become active ..
```

for a very long time after getting up, maybe we need to update also README with 

> git rm --cached -r .  # Remove every file from git's index.
> git reset --hard

**Comment by giovtorres** [comment]
Closed in #20 . Thanks for your contribution.
